### PR TITLE
feat: add release channel support for pulsar cluster

### LINKED ISSUE #43: [feat]: support release channel on terraform provider

### PR DESCRIPTION
**Pull request by urfreespace**
fix #43

### CODE REVIEW

**Review by tuteng — COMMENTED** [review]
Please run `go generate ./...` for update docs

File: cloud/resource_pulsar_cluster.go [hunk 1]
```diff
@@ -84,6 +84,13 @@ func resourcePulsarCluster() *schema.Resource {
 				Description:  descriptions["location"],
 				ValidateFunc: validateNotBlank,
 			},
+			"release_channel": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "rapid",
```
Comment (tuteng) [inline cloud/resource_pulsar_cluster.go:90]: Is the rapid channel used by default?

Comment (ericsyh) [reply cloud/resource_pulsar_cluster.go:90]: yes, on UI also uses the rapid as default

File: cloud/resource_pulsar_cluster.go [hunk 2]
```diff
@@ -84,6 +84,13 @@ func resourcePulsarCluster() *schema.Resource {
...
+				Description:  descriptions["release_channel"],
+				ValidateFunc: validateNotBlank,
```
Comment (tuteng) [inline cloud/resource_pulsar_cluster.go:92]: Maybe you can add a function `validateReleaseChannel` to verify if it is lts, rapid or null.

File: cloud/resource_pulsar_cluster.go (hunk 1 above)
Comment (urfreespace) [reply cloud/resource_pulsar_cluster.go:90]: yes, keep consistent with console UI

File: cloud/resource_pulsar_cluster.go (hunk 2 above)
Comment (urfreespace) [reply cloud/resource_pulsar_cluster.go]: fixed, thanks.

**Comment by urfreespace** [comment]
> Please run `go generate ./...` for update docs

done, thanks.

**Review by tuteng — APPROVED** [review]

**Pull request merged**